server/services/action.js

```diff
@@ -112,7 +112,7 @@ const actions = {
                     case actionTypes.BULK_CREATE_CASE: {
                         const { data: documentTags } = await getDocumentTags(context);
                         response = await createCase('/case/bulk', { caseType: context, form }, documentTags[0], headers);
-                        clientResponse = { summary: `Created ${response.data.count} new case${response.data.count > 1 ? 's' : ''}` };
+                        clientResponse = { child: { summary: `Created ${response.data.count} new case${response.data.count > 1 ? 's' : ''}` } };
```

Comment:
should `summary` on this line be changed to `title` here too? And have no `child` object, or an empty one? That would seem more consistent with the other changes I think

Reply:
Good catch there - I've added better handling of nonexistant child in the Panel.